test/bulk2.test.ts

```diff
@@ -6,7 +6,8 @@ import ConnectionManager from './helper/connection-manager';
 import config from './config';
 import { isObject, isString } from './util';
 import { isNodeJS } from './helper/env';
-import { BulkOperation, IngestJobV2Results } from 'jsforce/lib/api/bulk';
+import { BulkOperation } from 'jsforce/lib/api/bulk';
```

Comment:
I can't remember exactly if it's needed (don't think browser, jest gave me some problems a while ago and there's a module mapper so that `jsforce/` works).